urls.py

```diff
@@ -30,4 +30,14 @@
         views.show_offer,
         name='show_offer'
     ),
+    url(
```

Comment:
I think here should be some [organization] ID to distinct between all organizations.